### ISSUE TITLE
unable to creat headline in brain .org files

### ISSUE THREAD

**Issue by zerodel** [Kungsgeten/org-brain#92]
invoke org-brain-visualize won't work . it does not into that "Brain" interface . 
it did create the org file , but file is empty .  

messages be like. 
> 12 unique files scanned for IDs
> org-back-to-heading: Before first headline at position 1 in buffer  *temp*
> 
> 

**emacs version** info : 
GNU Emacs **25.3.1** (x86_64-apple-darwin13.4.0, NS appkit-1265.21 Version 10.9.5 (Build 13F1911))
 of 2017-09-13

**org version** : 
org-version is a variable defined in ‘org.el’.
Its value is "**9.1.9**"

**Comment by Kungsgeten** [comment]
That's strange. `org-back-to-heading` isn't used in `org-brain` directly. Could you please run `M-x toggle-debug-on-error`, which should give a more detailed description when the error occurs?

**Comment by zerodel** [comment]
thanks for your reply , and I fixed it with your advice. 

I followed the tutorial, and after that I remove those 'toy' .org files under brain folder to start a brand new brain entry collection , but there is still a ".org-brain-data.el" like this 

```
(setq org-brain-pins
      '(
	("graph_CS" "edge_CS" "A8E20DA0-3F4D-4590-88E8-00F34DAB5A8B")
	"dfs"
	))

```
 I guess it contains the pinned entry history , so next time I want to make entries,  org-brain can not find those files. 
everything seems OK after removing this hidden file.  

thanks again for your tool. it is very great indeed . the first time I meet org-mode, I wander: this head line structure is very alike the Brain's entry . just add the property indicates the graph relation . 
but I fail to implement it,  good work . man .
